gammapy/data/gti.py

```diff
@@ -107,6 +107,12 @@ def read(cls, filename, hdu="GTI"):
         table = Table.read(filename, hdu=hdu)
         return cls(table)
 
+    def to_table_hdu(self, name="GTI", format='gadf-dl3'):
```

Comment:
In other places we use `hdu_name` maybe change for consistency?

Reply:
I cannot find any method with a `hdu_name` argument in gammapy. All the `to_table_hdu` methods now have `name=` or do not support giving a name (should be changed?):

```
❯ rg 'def to_table_hdu'
gammapy/data/gti.py
110:    def to_table_hdu(self, name="GTI", format='gadf-dl3'):

gammapy/data/event_list.py
105:    def to_table_hdu(self, name="EVENTS", format='gadf-dl3'):

gammapy/maps/axes.py
1096:    def to_table_hdu(self, format="ogip"):
1803:    def to_table_hdu(self, format="gadf", hdu_bands=None):

gammapy/irf/core.py
488:    def to_table_hdu(self, format="gadf-dl3"):
```

Reply:
Also, we should probably replace `gadf-dl3` with `gadf`, right? `DL3` is not needed, since it is clear from what is being stored.

Reply:
Yes, you are right the argument e.g., used in `gammapy.maps` is just called `hdu` not `hdu_name` (that was probably wishful thinking of a better argument name...).

Concerning the `gadf` vs. `gadf-dl3`. There is one place where there is a distinction between these formats. A `MapAxis` can be serialized to a table both in `gadf` (DL4) and `gadf-dl3` (DL3) format However this distinction is only needed on the lowest level of `MapAxis` (which users basically never use independently...). It is still convenient to just forward the format specification, but I agree that on the high level of `IRF` and `Map` such is distinction is not needed...

Reply:
Conclusion from the dev meeting: only specify `format="gadf"` on the high level classes, such as `Observation`, `Map`, `IRF` etc. and then internally for `MapAxis` do the correct translation. Long-term the code on `MapAxis` will move to `Reader` and `Writer` classes.

Reply:
On second thought: I think we should remove the `name` argument completely. While there is no specification on the extname, we should be strict and just write consistent names from Gammapy to avoid a proliferation of HDU names in files written by Gammapy...if users really want they can always change the name later based on `BinTableHDU`...

Reply:
On second thought: I think we should remove the `name` argument completely. While there is no specification on the extname, we should be strict and just write consistent names from Gammapy to avoid a proliferation of HDU names in files written by Gammapy...if users really want they can always change the name later based on `BinTableHDU`...

Reply:
done